x11-wm/mutter/mutter-47.4.ebuild

```diff
@@ -0,0 +1,235 @@
+	SRC_URI=""
+	SLOT="0/14" # This can get easily out of date, but better than 9967
+else
+	KEYWORDS="~amd64 ~arm ~arm64 ~loong ~riscv ~x86"
+	SLOT="0/$(($(ver_cut 1) - 32))" # 0/libmutter_api_version - ONLY gnome-shell (or anything using mutter-clutter-<api_version>.pc) should use the subslot
+fi
+
+IUSE="debug elogind gnome gtk-doc input_devices_wacom +introspection +libdisplay screencast sysprof systemd test udev wayland x11 video_cards_nvidia"
+# native backend requires gles3 for hybrid graphics blitting support, udev and a logind provider
+REQUIRED_USE="
+	gtk-doc? ( introspection )
+	wayland? ( ^^ ( elogind systemd ) udev )
+	test? ( wayland )"
+RESTRICT="!test? ( test )"
+
+# gnome-settings-daemon is build checked, but used at runtime only for org.gnome.settings-daemon.peripherals.keyboard gschema
+# USE=libei was first introduced in xwayland-23.2.1; we min dep on that to ensure the [libei(+)] works right, as missing USE flag with
+# previous versions meant that it's not there, while the intention seems to be to make it always enabled without USE flag in the future;
+# this ensures have_enable_ei_portal is always there in xwayland.pc, which affects how Xwayland is launched, thus if it were toggled off
+# in Xwayland after mutter is installed, Xwayland would fail to be started by mutter. mutter already hard-depends on libei, so there's
+# really no extra deps here (besides xdg-desktop-portal, but we want that too, anyhow).
+# v3.32.2 has many excessive or unused *_req variables declared, thus currently the dep order ignores those and goes via dependency() call order
+DEPEND="
+	>=media-libs/graphene-1.10.2[introspection?]
+	x11-libs/gdk-pixbuf:2
+	>=x11-libs/pango-1.46[introspection?]
+	>=x11-libs/cairo-1.14[X]
+	>=x11-libs/pixman-0.42
+	>=dev-libs/fribidi-1.0.0
+	>=gnome-base/gsettings-desktop-schemas-47_beta[introspection?]
+	>=dev-libs/glib-2.81.1:2
+	gnome-base/gnome-settings-daemon
+	>=dev-libs/json-glib-0.12.0[introspection?]
+	>=x11-libs/libxkbcommon-0.4.3
+	x11-libs/libICE
+	>=app-accessibility/at-spi2-core-2.46:2[introspection?]
+	sys-apps/dbus
+	>=x11-misc/colord-1.4.5:=
+	>=media-libs/lcms-2.6:2
+	>=media-libs/harfbuzz-2.6.0:=
+	>=dev-libs/libei-1.0.901
+
+	gnome? ( gnome-base/gnome-desktop:4= )
+
+	>=media-libs/libcanberra-0.26
+
+	media-libs/libglvnd[X]
+
+	wayland? (
+		>=dev-libs/wayland-protocols-1.32
+		>=dev-libs/wayland-1.22.0
+
+		>=x11-libs/libdrm-2.4.118
+		media-libs/mesa[gbm(+)]
+		>=dev-libs/libinput-1.26.0:=
+
+		elogind? ( sys-auth/elogind )
+		>=x11-base/xwayland-23.2.1[libei(+)]
+		video_cards_nvidia? ( gui-libs/egl-wayland )
+	)
+	udev? (
+		>=virtual/libudev-232-r1:=
+		>=dev-libs/libgudev-238
+	)
+	systemd? ( sys-apps/systemd )
+	x11-libs/libSM
+	input_devices_wacom? ( >=dev-libs/libwacom-0.13:= )
+	>=x11-libs/startup-notification-0.7
+	screencast? ( >=media-video/pipewire-1.2.0:= )
+	introspection? ( >=dev-libs/gobject-introspection-1.54:= )
+	libdisplay? ( media-libs/libdisplay-info )
```

Comment:
You broke this!!!

Before this PR, libdisplay-info was an unconditional dependency that used a `media-libs/libdisplay-info:=` dependency. It was added in 7d2273a9e780199b7a28bd06d17234130bc469e5, you rebased against this commit but didn't incorporate the changes. I could maybe understand keeping the USE flag but dropping this from `:=` to an unqualified dependency?